src/styles/base/_typography.scss

```diff
@@ -1,6 +1,7 @@
 // Robot Condensed:
 @import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500);
 @import url(https://fonts.googleapis.com/css?family=Roboto+Condensed);
+@import url(https://fonts.googleapis.com/css?family=Lora:400,400italic);
```

Comment:
Be careful with font creep, they can really slow down a page load if you have too many.